### PR TITLE
some icp additon repo

### PR DESCRIPTION
**Pull request by hrusikesh8280**

### CODE REVIEW

File: data/ecosystems/i/internet-computer.toml [hunk 1]
```diff
@@ -21728,3 +21728,205 @@ url = "https://github.com/NnsDao/nnsdao-fe"
...
+url = "https://github.com/GMDancun/Tractor-management-system"
+
+[[repo]]
+url = "https://github.com/edwinaloo/votingapp"
+
+[[repo]]
+url = "https://github.com/ayeritedw/App_Classroom"
+
+[[repo]]
+url = "https://github.com/DevKipesa/client"
```
Comment (eherrerosj) [inline data/ecosystems/i/internet-computer.toml:21756]: how's this using ICP?

File: data/ecosystems/i/internet-computer.toml [hunk 2]
```diff
@@ -21728,3 +21728,205 @@ url = "https://github.com/NnsDao/nnsdao-fe"
 [[repo]]
 url = "https://github.com/michi-labs/d-snap"
 
+[[repo]]
+url = "https://github.com/GabrielPlus/Delicious"
```
Comment (eherrerosj) [inline data/ecosystems/i/internet-computer.toml:21732]: how's this using ICP?

**Review by eherrerosj — CHANGES_REQUESTED** [review]
left a comments

**Comment by hrusikesh8280** [comment]
i have reviewed the repos again and removed suggested changes, thanks for giving it a thorough look. Hope this helps

File: data/ecosystems/i/internet-computer.toml [hunk 3]
```diff
@@ -21726,7 +21726,161 @@ url = "https://github.com/NeutronStarDAO/Proton"
...
+url = "https://github.com/PetuchoDev/PTFG"
+
+[[repo]]
+url = "https://github.com/EddieTPC/CryptoCrowd"
+
+[[repo]]
+url = "https://github.com/ChristianDelRalba/ProyectoICP"
+
+[[repo]]
+url = "https://github.com/Glinrear/Mavain/tree/main/identity_system/canisters/identity"
```
Comment (eherrerosj) [inline data/ecosystems/i/internet-computer.toml:21819]: this repo URL format is incorrect

File: data/ecosystems/i/internet-computer.toml [hunk 4]
```diff
@@ -21726,7 +21726,161 @@ url = "https://github.com/NeutronStarDAO/Proton"
...
+url = "https://github.com/riokum27/VistaWeather"
+
+[[repo]]
+url = "https://github.com/Joey-mi/qr-verifier-verson2"
+
+[[repo]]
+url = "https://github.com/KaleF07/recipe-book"
+
+[[repo]]
+url = "https://github.com/Hexdee/OpenBridge"
```
Comment (eherrerosj) [inline data/ecosystems/i/internet-computer.toml:21795]: how's this repo using ICP?

File: data/ecosystems/i/internet-computer.toml [hunk 5]
```diff
@@ -21726,7 +21726,161 @@ url = "https://github.com/NeutronStarDAO/Proton"
...
+
+[[repo]]
+url = "https://github.com/mariogarcia-ar/tco-icp-sales-chatbot/"
+
+[[repo]]
+url = "https://github.com/ArielRobotti/ICP_Freelancer"
+
+[[repo]]
+url = "https://github.com/SistemasTecTlaxiaco/cafeteriaMagnate1"
```
Comment (eherrerosj) [inline data/ecosystems/i/internet-computer.toml:21804]: this one belongs to NEAR I believe

**Review by eherrerosj — CHANGES_REQUESTED** [review]
left a few more comments. Can you please make sure that the rest of the URLs are well formatted and that are repos that are actually using ICP please?

**Comment by hrusikesh8280** [comment]
> left a few more comments. Can you please make sure that the rest of the URLs are well formatted and that are repos that are actually using ICP please?
@eherrerosj , I have checked and corrected the format and removed unnecessary repos.

**Comment by jubos** [comment]
Manually merged here [e6e0949](https://github.com/electric-capital/crypto-ecosystems/commit/e6e094980cd7161915b9fff5186915a5278060ca)

**Pull request closed without merging**